modules/main-rules/pages/attempt-repairs.adoc

```diff
@@ -1,11 +1,11 @@
 = R3. Attempt repairs
 
-This step is relevant only for war engines with minor (repairable) damage or collapsed shields.
+This step is relevant only for war engines with minor (repairable) damage or collapsed void shields.
 
-For each damaged location and each collapsed shield, take a Repair test.
+For each repairable critical result and each collapsed shield, take a Repair test.
```

Comment:
```suggestion
For each repairable critical result and each collapsed void shield, take a Repair test.
```

Just spotted a 'void' that I missed.

Reply:
Very slight additional tweaks to be semantically precise.

```suggestion
For each repairable critical damage effect and each collapsed void shield, take a Repair test.
```

Reply:
Looks good.